### PR TITLE
Mergify/bp/rolling/pr 386

### PR DESCRIPTION
**Pull request by Zard-C**
Hi, @JanStaschulat, I am trying to fix the doi check problem in pr-386

### CODE REVIEW

**Comment by JanStaschulat** [comment]
Hi @Zard-C, could you summarize what is the difference of this pull request compared to PR https://github.com/ros2/rclc/pull/387?

**Comment by Zard-C** [comment]
> Hi @Zard-C, could you summarize what is the difference of this pull request compared to PR #387?

Thanks @JanStaschulat , I just saw you comment. There's no difference in code, I just want to pass the DOI check, and I will close this PR. Thanks a lot  :)
